ingestion/src/metadata/ingestion/api/source.py

```diff
@@ -9,10 +9,12 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import time
```

Comment:
using `from time import perf_counter` is usually a better option for measuring performance

Reply:
https://www.geeksforgeeks.org/time-perf_counter-function-in-python/